test-data/unit/semanal-errors.test

```diff
@@ -3,8 +3,7 @@ in 1
 def f():
   1 1
 [out]
-main:1: error: Parse error before in
-main:3: error: Parse error before numeric literal
```

Comment:
Actually, the test should probably just be deleted.  It looks like it's supposed to test multiple syntax errors, which we will no longer display.